modes/assets/semeru/Dockerfile.fights.semeru

```diff
@@ -66,7 +66,7 @@ COPY --chown=185 target/quarkus-app/quarkus/ /deployments/quarkus/
 
 EXPOSE 8083
 
-ENV JAVA_OPTS_APPEND="-Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager"
+ENV JAVA_OPTS_APPEND="-Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager -XX:+UseJITServer -XX:JITServerAddress=${SUT_SERVER}"
```

Comment:
I see this `-XX:+UseJITServer -XX:JITServerAddress=${SUT_SERVER}` is always set, even if we don't start the JIT server.. is it expected? I was thinking that it would be better to add such options only if we are enabling (and starting up the JIT server). wdyt?

Reply:
If it cant find the JITserver, it just behaves as normal, so I think this is ok to just leave in.

Reply:
sounds good then!